core/DataTable/Row.php

```diff
@@ -748,25 +750,27 @@ public static function isEqual(Row $row1, Row $row2)
     private function warnIfSubtableAlreadyExists()
     {
         if (!is_null($this->subtableId)) {
-            Log::warning(
+            $ex = new \Exception(sprintf(
                 "Row with label '%s' (columns = %s) has already a subtable id=%s but it was not loaded - overwriting the existing sub-table.",
                 $this->getColumn('label'),
                 implode(", ", $this->getColumns()),
                 $this->getIdSubDataTable()
-            );
+            ));
+            StaticContainer::get(LoggerInterface::class)->warning("{exception}", ['ex' => $ex]);
         }
     }
 
     protected function warnWhenSummingTwoStrings($thisColumnValue, $columnToSumValue, $columnName = null)
     {
         if (is_string($columnToSumValue)) {
-            Log::warning(
+            $ex = new \Exception(sprintf(
                 "Trying to add two strings in DataTable\Row::sumRowArray: %s + %s for column %s in row %s",
                 $thisColumnValue,
                 $columnToSumValue,
                 $columnName,
                 $this->__toString()
-            );
+            ));
+            Log::warning("{exception}", ['ex' => $ex]);
```

Comment:
Any reason you changed that to using `StaticContainer` above, but not here? Might be good to have that consitent

Reply:
no reason, I just missed that entirely